src/templates/dto/dto.template.ts

```diff
@@ -1,5 +1,5 @@
 import { SwaggerApi } from '@swaggertypes/documentSwagger.type'
-import { allOfDereference } from '@templates/dto/allOfdereference'
+import { allOfDereference } from '@templates/lib/allOfdereference'
```

Comment:
```suggestion
import { allOfDereference } from '@templates/dto'
```

здесь такая запись может сработать, так как функция в index.ts выводится в templates/dto

Reply:
исправлено